tests/integration/results.go

```diff
@@ -123,8 +123,12 @@ func (ucid *UniqueCid) Validate(s *state, actualValue any, msgAndArgs ...any) {
 
 	if isNew {
 		require.IsType(s.t, "", actualValue)
+		value, ok := actualValue.(string)
+		if !ok {
+			require.Fail(s.t, "UniqueCid actualValue string cast failed")
```

Comment:
suggestion: I think I prefer a `nolint` to this, this is dead code and makes reading the func more complicated than it needs to be

Reply:
And if you really really don't want a no lint, please remove the `require.IsType` call on the line above

Reply:
One can argue reading any go `if err != nil {` makes the code unreadable (which i agree it does, but we don't stop using it). I don't like the linter suppressing suggestion keeping as is.

- [x] Remove `require.IsType`